src/app/components/editors/ck-editor/ck-editor-full.directive.ts

```diff
@@ -0,0 +1,13 @@
+import { Directive } from '@angular/core';
+import { CKEditorComponent } from 'ckeditor4-angular';
+
+@Directive({
+    selector: '[tcCkFull]'
+})
+export class CKEditorFullDirective
```

Comment:
Wollten wir sie nicht einfach CkEditorDirective nennen?
```suggestion
export class CKEditorDirective
```